docs/user/tutorials/01-60-security/01-60-tls-certificates-security.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Set up or update a custom domain TLS certificate
+---
+
+The TLS certificate is a vital security element. Follow this tutorial to set up your custom TLS certificate in Kyma or update a manually rotated one.
+
+>**NOTE:** This procedure can interrupt the communication between your cluster and the outside world for a limited period of time.
+
+## Prerequisites
+
+- Generate a new TLS certificate and key for custom domain deployments, base64-encoded. If you want to update your custom TLS certificate in Kyma, rotate or renew it manually.
+- Generate a `kubeconfig` file for the Kubernetes cluster that hosts the Kyma instance.
+
+## Steps
+
+1. Export your domain, new or rotated certificate, and key as the environment variables.
+
+```bash
+export DOMAIN={YOUR_DOMAIN}
+export TLS_CERT={YOUR_NEW_CERTIFICATE}
+export TLS_KEY={YOUR_NEW_KEY}
+```
+
+2. Deploy Kyma with your custom domain certificate. Run:
```

Comment:
We should adjust this step to modular Kyma installation, right?

Reply:
Deploying Kyma with generated certificate will no longer be possible after modularisation. This tutorial is deprecated